frontend/app/routes/_gcweb-app.view-letters._index.tsx

```diff
@@ -17,10 +17,12 @@ export const handle = {
 } as const satisfies RouteHandleData;
 
 // TODO Create loader function to fetch data from mock using service
+export async function loader({ request }: LoaderFunctionArgs) {
```

Comment:
Loader function is reponsible to fetch the data with sorting and paging. This is done on the server side. Client component will render based on the loader returned state (with useLoaderData).